packages/react-scripts/template/README.md

```diff
@@ -434,7 +434,7 @@ REACT_APP_SECRET_CODE=abcdef npm start
 ```
 
 > Note: Defining environment variables in this manner is temporary for the life of the shell session. Setting
-permanent environment variables is outside the scope of these docs.
+permanent environment variables in development can be done in a .env file in the root of your project.
```

Comment:
This should link to dotenv documentation.
Also please use backticks for `.env` so it’s highlighted.